scripts/changelog-script.sh

```diff
@@ -20,42 +12,23 @@ fi
 
 MIRROR="https://mirror.openshift.com/pub/openshift-v4/clients/odo/$2/"
 INSTALLATION_GUIDE="https://docs.openshift.com/container-platform/latest/cli_reference/developer_cli_odo/installing-odo.html"
+GIT_TREE="https://github.com/openshift/odo/tree/$2"
+FULL_CHANGELOG="https://github.com/openshift/odo/compare/$1...$2"
+
+echo -e "
+# Release of $2
+
+## [$2]($GIT_TREE) ($(date '+%Y-%m-%d'))
 
-echo -e "# Installation of $2
+[Full Changelog]($FULL_CHANGELOG)
+
+-- COPY CONTENT FROM Changelog.md here --
+
+# Installation of $2
 
 To install odo, follow our installation guide at [docs.openshift.com]($INSTALLATION_GUIDE)
 
-After each release, binaries are synced to [mirror.openshift.com]($MIRROR)" > /tmp/base
-
-github_changelog_generator \
---max-issues 500 \
---user openshift \
---project odo \
--t $GITHUB_TOKEN \
---since-tag $1 \
---future-release $2 \
---base /tmp/base \
---output /tmp/changelog \
---exclude-tags "lifecycle/rotten" \
---header-label "# Release of $2" \
---enhancement-label "**Code Refactoring:**" \
---enhancement-labels "kind/cleanup,kind/api-change,kind/code-refactoring" \
---breaking-label "**New features:**" \
---breaking-labels "kind/feature" \
---bugs-label "**Bugs:**" \
---bug-labels "kind/bug" \
---removed-label "**Tests:**" \
---removed-labels "kind/flake,kind/test,kind/failing-test" \
---security-label "**Documentation & Discussions:**" \
---security-labels "kind/docs,kind/documentation,kind/design,kind/discussion,kind/epic"
-
-RESULT=$?
-
-if [ $RESULT -eq 0 ]; then
-  echo ""
-  echo "The changelog is located at: /tmp/changelog"
-  echo ""
-else
-  echo -e "Unable to generate changelog using github_changelog_generator"
-  exit 1
-fi
+After each release, binaries are synced to [mirror.openshift.com]($MIRROR)" > /tmp/changelog
+
+echo "The changelog is located at: /tmp/changelog"
+echo ""
```

Comment:
Can we **also** print the contents of `/tmp/changelog` in the output itself if it is not too much work?

Reply:
I suggest this because I expected the script to do so when I ran it and instead it just printed the file location, so I was kinda disappointed when I saw the output :sweat_smile:

Reply:
I guess so.

Reply:
:) yea i guess this was not happening before either and the comment on top said so too